docs/sections/user_guide/cli/drivers/fv3.rst

```diff
@@ -44,8 +46,8 @@ All tasks take the same arguments. For example:
 .. code-block:: text
 
    $ uw fv3 run --help
-   usage: uw fv3 run --config-file PATH --cycle CYCLE [-h] [--batch] [--dry-run] [--debug] [--quiet]
```

Comment:
Thanks for catching the remnant `--debug`s in this update

Reply:
It will all be automated soon!